app/components/gallery/PictureItem.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <a href="#" @click.prevent="shadowbox = !shadowbox">
+    <div class="photo aspect-ratio-1/1">
+      <img class="image-fit" :src="photo.full_image_url" :srcset="photo.medium_srcset" :alt="photo.title">
+      <div class="description">
+        {{ photo.title }}
+      </div>
+    </div>
+    <Modal :media="photo.full_image_url" :title="photo.title" :visibility="shadowbox" />
+  </a>
+</template>
+
+<script>
+export default {
+  props: {
+    photo: {
+      type: Object,
+      required: true
```

Comment:
Could we define more exact props here that we use in the template? 

So is vue props here something like the parameters of an object during initialisation? In that case I think it would be clearest to here define `title` `full_image_url` etc. that are referenced in the template. Or is this already defined somewhere that I missed? Or did I go the wrong way? Curious to hear your thoughts! Thanks.